src/utils/consent.js

```diff
@@ -8,10 +8,13 @@ export const isConsentGranted = () => {
 
 export const grantConsent = () => {
   const version = appVersion()
-  sendEvent('KaiOSAppConsent', 20122102, getAppLanguage(), { version })
+  sendEvent('KaiOSAppFirstRun', 20158341, getAppLanguage(), {
+    app_version: version,
```

Comment:
From [the schema](https://meta.wikimedia.org/wiki/Schema:KaiOSAppFirstRun) I'm seeing that `app_id` comes first and `app_version` second, confirm the order doesn't matter?

Reply:
Order of the keys in a json doc doesn't matter